lms/djangoapps/learner_dashboard/tests/test_utils.py

```diff
@@ -0,0 +1,28 @@
+""" Unit test module covering utils module
+"""
+
+import ddt
+from django.test import TestCase
+
+from lms.djangoapps.learner_dashboard.utils import (
+    FAKE_COURSE_ID_FOR_URL,
+    url_remove_fake_course_id)
+
+
+@ddt.ddt
+class TestUrlRemoveFakeId(TestCase):
+    """ The test case class covering the util function 'url_remove_fake_course_id'
+    """
+    @ddt.data(['path1'], ['path1', 'path2'], [])
+    def test_remove_fake_id_from_url(self, path_array):
```

Comment:
@schenedx the utility should be tested, but you can do it more simply and without duplicating as much code:
```py
def test_strip_course_id(self):
    base = 'foo/'
    actual = strip_course_id(base + FAKE_COURSE_KEY)
    self.assertEqual(actual, base)
```